proprietary-specs.md

```diff
@@ -11,7 +11,7 @@ the responsibility of the respective project maintainers.
 
 - Create a spec that follows the structure of an existing binding specification (e.g. [http](http-protocol-binding.md) or [amqp](amqp-protocol-binding.md)) - this will help SDK development.
   - **NOTES:**
-    - The spec needs to be publically accessible and managed by the proposing organization.
-    - The spec should clearly state the version(s) of CloudEvents supported.
+    - The spec MUST be publically accessible and managed by the proposing organization.
+    - The spec MUST clearly state the version(s) of CloudEvents supported.
```

Comment:
+1 to "must" but let's lowercase them since these aren't bound by rfc2119

Reply:
Done